src/App.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Home from './components/Home';
+import Books from './components/Books';
+import Components from './components/Components';
+
+// end imports
+
+function App() {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/books" element={<Books />} />
+        <Route path="/components" element={<Components />} />
+      </Routes>
```

Comment:
[REQUIRED] You should have routes to home and categories page. But I can't see any route to categories page. Please fix this.